packages/synapse-react-client/src/components/entity/page/title_bar/EntityPageTitleBar.tsx

```diff
@@ -47,7 +47,16 @@ export default function EntityPageTitleBar(props: EntityPageTitleBarProps) {
           backgroundColor: 'grey.100',
         }}
       >
-        <Stack direction={'row'}>
+        <Stack
+          sx={theme => ({
+            [theme.breakpoints.down('sm')]: {
+              flexDirection: 'column',
+            },
+            [theme.breakpoints.up('sm')]: {
+              flexDirection: 'row',
+            },
+          })}
```

Comment:
Same but more concise
```suggestion
          sx={{
            flexDirection: { xs: 'column', sm: 'row' }
          })}
```